ui/app/AppLayouts/Profile/panels/ProfileShowcaseBasePanel.qml

```diff
@@ -0,0 +1,329 @@
+        hiddenListView.model = root.baseModel
+    }
+
+    signal showcaseEntryChanged()
+
+    QtObject {
+        id: d
+
+        readonly property int defaultDelegateHeight: 60
+        readonly property int contentSpacing: 12
+        readonly property int strokeMargin: 2
+        readonly property int shapeRectangleHeight: 48
+    }
+
+    clip: true
+
+    ScrollBar.vertical: StatusScrollBar {
+        policy: ScrollBar.AsNeeded
+        visible: resolveVisibility(policy, root.height, root.contentHeight)
+    }
+
+    flickable1: EmptyShapeRectangleFooterListView {
+        id: inShowcaseListView
+
+        model: root.showcaseModel
+        width: root.width
+        placeholderText: root.emptyInShowcasePlaceholderText
+        placeholderHeight: d.shapeRectangleHeight
+
+        header: FoldableHeader {
+            width: ListView.view.width
+            title: qsTr("In showcase")
+            folded: root.flickable1Folded
+
+            onToggleFolding: root.flip1Folding()
+        }
+
+        delegate: DropArea {
+            id: showcaseDelegateRoot
+
+            property int visualIndex: index
+
+            width: ListView.view.width
+            height: visible && showcaseDraggableDelegateLoader.item ? showcaseDraggableDelegateLoader.item.height : 0
+
+            keys: ["x-status-draggable-showcase-item"]
+            visible: model.showcaseVisibility !== Constants.ShowcaseVisibility.NoOne
+
+            onEntered: function(drag) {
+                const from = drag.source.visualIndex
+                const to = showcaseDraggableDelegateLoader.item.visualIndex
+                if (to === from)
+                    return
+                root.showcaseEntryChanged()
+                root.showcaseModel.move(from, to, 1)
+                drag.accept()
+            }
+
+            // TODO:
+            // This animation is causing issues when there are no elements in the showcase list.
+            // Reenable it once the refactor of the models and delegates is done (simplified): #13498
+            // ListView.onRemove: SequentialAnimation {
+            //  PropertyAction { target: showcaseDelegateRoot; property: "ListView.delayRemove"; value: true }
+            //      NumberAnimation { target: showcaseDelegateRoot; property: "scale"; to: 0; easing.type: Easing.InOutQuad }
+            //      PropertyAction { target: showcaseDelegateRoot; property: "ListView.delayRemove"; value: false }
+            // }
+
+            // In showcase delegate item container:
+            Loader {
+                id: showcaseDraggableDelegateLoader
+                width: parent.width
+                sourceComponent: root.showcaseDraggableDelegateComponent
+
+                property var modelData: model
+                property var dragParentData: root
+                property int visualIndexData: index
+            }
+        }
+
+        // TO BE REDEFINED (task #13509): Overlaid at the bottom of the listview
+        DropArea {
```

Comment:
The drop areas need be disabled if the listviews are collapsed. We have ghost drop areas now 😄

Reply:
Ok! I'll take a look. I didn't take a lot of care to the drop areas tbh since we need to work on them again in the next iteration. I'll try to remove the ghosts :grimacing:

Reply:
Will be covered in here: https://github.com/status-im/status-desktop/issues/13509